### ISSUE TITLE
getAnnotationLabels to retrieve previous/next token annotation labels

### ISSUE THREAD

**Issue by robertfromont** [nzilbb/labbcat-R#8]
Add a `token` parameter to allow retrieval of previous (-1) or next (1) token annotation labels.

e.g. to get the orthography of the word before the match:
```
previous.word <- getAnnotationLabels(labbcat.url, results$MatchId, "orthography", token=-1)
```

**Comment by robertfromont** [comment]
Make sure that token can be -2, 3, etc. (and if should default to 0)

**Comment by robertfromont** [comment]
Also make sure that the layer ID passed intelligently figures out at what scope to decide what 'next' means.  e.g. if the MatchId identifies a phone ("segment")...

- but the layer ID is a word layer like "orthography" or "POS", then it should be the label of the next _word_
- but if the layer ID is a segment layer like "segments", then it should be the label of the next _segment_
- (and if it's a 'turn' layer, I guess it should be the label of the next turn, probably which isn't useful, but is consistent)

**Comment by robertfromont** [comment]
Also add an option for including start/end time as well as label

**Comment by robertfromont** [comment]
This will be released in 0.5-1
